src/esoh/ingest/messages.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+import xarray as xr
+
+import uuid
+
+from esoh.ingest.netCDF.extract_metadata_netcdf import build_all_json_payloads_from_netCDF
+
+
+def build_message(file: [object], input_type: str, uuid_prefix: str):
+    match input_type:
```

Comment:
same here

Reply:
The reason I chose to use this here is for ease of adding more filetypes at a later stage, and then making it easier to read when this list gets longer. I do agree that when there is only one filetype implemented there is no need to use match-case